### PR TITLE
Task/hw 58540 update ui sdk tests to fix i os 13 updates

### PR DESCRIPTION
**Pull request by asenthil-altmtrk**

### CODE REVIEW

File: UITests/TransferFunds/TransferUserFundsTest.swift [hunk 1]
```diff
@@ -17,6 +19,11 @@ class TransferUserFundsTest: BaseTests {
         transferFunds = TransferFunds(app: app)
         selectDestination = TransferFundsSelectDestination(app: app)
         addTransferMethod = AddTransferMethod(app: app)
+        if #available(iOS 13.0, *) {
+        elementQuery = app.tables["scheduleTransferTableView"].buttons
+        } else {
+        elementQuery = app.tables["scheduleTransferTableView"].staticTexts
```
Comment (jsaini-hw) [inline UITests/TransferFunds/TransferUserFundsTest.swift]: indentation?

File: UITests/TransferFunds/TransferUserFundsTest.swift [hunk 2]
```diff
@@ -343,9 +350,15 @@ class TransferUserFundsTest: BaseTests {
         XCTAssertEqual(selectDestination.getSelectDestinationRowDetail(index: 2), "United States Ending on 4281")
 
         // Assert first row is checked by default
+        if #available(iOS 13.0, *) {
+        XCTAssertTrue(usdBankAccount.buttons["checkmark"].exists, "By default the first row should be selected")
+        XCTAssertFalse(cadBankAccount.buttons["checkmark"].exists, "By default the first row should be selected")
+        XCTAssertFalse(prepaidCard.buttons["checkmark"].exists, "By default the first row should be selected")
```
Comment (jsaini-hw) [inline UITests/TransferFunds/TransferUserFundsTest.swift]: indentation

File: UITests/TransferFunds/TransferUserFundsTest.swift [hunk 3]
```diff
@@ -343,9 +350,15 @@ class TransferUserFundsTest: BaseTests {
...
+        } else {
         XCTAssertTrue(usdBankAccount.buttons["More Info"].exists, "By default the first row should be selected")
         XCTAssertFalse(cadBankAccount.buttons["More Info"].exists, "By default the first row should be selected")
         XCTAssertFalse(prepaidCard.buttons["More Info"].exists, "By default the first row should be selected")
```
Comment (jsaini-hw) [inline UITests/TransferFunds/TransferUserFundsTest.swift]: indentation

File: UITests/TransferMethod/AddTransferMethodBankAccountIndividualTests.swift [hunk 4]
```diff
@@ -26,6 +26,11 @@ class AddTransferMethodBankAccountIndividualTests: BaseTests {
         spinner = app.activityIndicators["activityIndicator"]
         waitForNonExistence(spinner)
         addTransferMethod = AddTransferMethod(app: app)
+        if #available(iOS 13.0, *) {
+        elementQuery = app.tables["addTransferMethodTable"].buttons
+        } else {
+        elementQuery = app.tables["addTransferMethodTable"].staticTexts
+        }
```
Comment (jsaini-hw) [inline UITests/TransferMethod/AddTransferMethodBankAccountIndividualTests.swift]: indentation

File: UITests/TransferMethod/AddTransferMethodPayPalAccountTests.swift [hunk 5]
```diff
@@ -25,6 +25,11 @@ class AddTransferMethodPayPalAccountTests: BaseTests {
         spinner = app.activityIndicators["activityIndicator"]
         waitForNonExistence(spinner)
         addTransferMethod = AddTransferMethod(app: app)
+        if #available(iOS 13.0, *) {
+        elementQuery = app.tables["addTransferMethodTable"].buttons
+        } else {
+        elementQuery = app.tables["addTransferMethodTable"].staticTexts
+        }
```
Comment (jsaini-hw) [inline UITests/TransferMethod/AddTransferMethodPayPalAccountTests.swift]: indentation

File: UITests/TransferMethod/AddTransferMethodWireAccountTests.swift [hunk 6]
```diff
@@ -26,6 +26,11 @@ class AddTransferMethodWireAccountIndividualTests: BaseTests {
         spinner = app.activityIndicators["activityIndicator"]
         waitForNonExistence(spinner)
         addTransferMethod = AddTransferMethod(app: app)
+        if #available(iOS 13.0, *) {
+        elementQuery = app.tables["addTransferMethodTable"].buttons
+        } else {
+        elementQuery = app.tables["addTransferMethodTable"].staticTexts
+        }
```
Comment (jsaini-hw) [inline UITests/TransferMethod/AddTransferMethodWireAccountTests.swift]: indentation

File: UITests/BaseTests/BaseTest.swift [hunk 7]
```diff
@@ -4,6 +4,7 @@ class BaseTests: XCTestCase {
     var app: XCUIApplication!
     var mockServer: HyperwalletMockWebServer!
     var spinner: XCUIElement!
+    var table: XCUIElement!
```
Comment (jsaini-hw) [inline UITests/BaseTests/BaseTest.swift]: where is this used?

File: UITests/Helper/TransferFunds/TransferFundsConfirmation.swift [hunk 8]
```diff
@@ -58,7 +58,11 @@ class TransferFundsConfirmation {
         foreignExchangeBuy = scheduleTable.staticTexts["transfer_fx_buy_confirmation".localized()]
         foreignExchangeRate = scheduleTable.staticTexts["transfer_fx_rate_confirmation".localized()]
 
+        if #available(iOS 13.0, *) {
+        confirmButton = scheduleTable.cells.buttons["scheduleTransferLabel"]
```
[the remark this answers is not in the document]
Comment (asenthil-altmtrk) [reply UITests/Helper/TransferFunds/TransferFundsConfirmation.swift]: indentation added

File: UITests/TransferFunds/TransferUserFundsTest.swift (hunk 1 above)
Comment (asenthil-altmtrk) [reply UITests/TransferFunds/TransferUserFundsTest.swift]: indentation added

File: UITests/TransferFunds/TransferUserFundsTest.swift (hunk 2 above)
Comment (asenthil-altmtrk) [reply UITests/TransferFunds/TransferUserFundsTest.swift]: indentation added

File: UITests/TransferFunds/TransferUserFundsTest.swift (hunk 3 above)
Comment (asenthil-altmtrk) [reply UITests/TransferFunds/TransferUserFundsTest.swift]: indentation added

File: UITests/TransferMethod/AddTransferMethodBankAccountIndividualTests.swift (hunk 4 above)
Comment (asenthil-altmtrk) [reply UITests/TransferMethod/AddTransferMethodBankAccountIndividualTests.swift]: indentation added

File: UITests/TransferMethod/AddTransferMethodWireAccountTests.swift (hunk 6 above)
Comment (asenthil-altmtrk) [reply UITests/TransferMethod/AddTransferMethodWireAccountTests.swift]: indentation added

File: UITests/BaseTests/BaseTest.swift (hunk 7 above)
Comment (asenthil-altmtrk) [reply UITests/BaseTests/BaseTest.swift]: removed

**Pull request merged**